handlers/messagebird/handler.go

```diff
@@ -62,14 +62,16 @@ var statusMapping = map[string]courier.MsgStatus{
 	"expired":         courier.MsgStatusFailed,
 }
 
-type ReceivedMessage struct {
-	ID              string   `json:"id"`
-	Recipient       string   `json:"recipient"`
-	Originator      string   `json:"originator"`
-	Body            string   `json:"body"`
-	CreatedDatetime string   `json:"createdDatetime"`
-	MediaURLs       []string `json:"mediaUrls"`
-	MMS             bool     `json:"mms"`
+type formMessage struct {
+	ID              string   `name:"id"`
+	MID             string   `name:"mid"`       //shortcode only
+	Shortcode       string   `name:"shortcode"` //shortcode only
+	Recipient       string   `name:"recipient" validate:"required"`
+	Originator      string   `name:"originator"`
+	Body            string   `name:"body"`
+	MediaURLs       []string `name:"mediaUrls"`
+	MessageBody     string   `name:"message"` //shortcode only
+	CreatedDatetime string   `name:"receive_datetime" validate:"required"`
```

Comment:
We need a different field for the date time `createdDatetime ` for long numbers  and `receive_datetime` for short codets , we can remove the required tag 

https://developers.messagebird.com/api/sms-messaging/#receive-and-handle-inbound-sms

Reply:
ah yes, I need to fix that- this is the annoying part where they name the fields differently between shortcodes and long numbers